docs/DeveloperGuide.md

```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
           Use case ends.
 
+**Use case: Check eligibility for assistance**
```

Comment:
For the use cases I updated in my PR I added till `UC05` so might want to put `UC06` onwards for ur use cases to standardise